src/components/button/button.styles.ts

```diff
@@ -0,0 +1,56 @@
+import appTheme from 'boilerplate-react-native/src/app-theme';
+import { StyleSheet } from 'react-native';
+
+export const styles = StyleSheet.create({
+  button: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    minHeight: 40,
+    borderWidth: 1,
+    borderRadius: appTheme.radii?.md || 8,
+    width: '100%',
+    gap: appTheme.space?.['2'] || 8,
```

Comment:
I see space, radii, as `?`. JS hides the complexity behind simple `?`, but what its implying is it can be null or undefined.

Ideally that should never the be the case and we dont have to put a fallback.

Is that something we have to do at app setup and use `!` instead of `?`